core/src/test/java/tech/pegasys/eth2signer/core/multikey/BlsArtifactSignerFactoryTest.java

```diff
@@ -93,7 +114,8 @@ void createsArtifactSignerFromKeyStoreUsingAbsolutePaths() {
 
     assertThat(keystoreFile).isAbsolute();
     assertThat(passwordFile).isAbsolute();
-    assertThat(artifactSigner.getIdentifier()).isEqualTo("0x" + PUBLIC_KEY);
+    assertThat(artifactSigner.getIdentifier())
```

Comment:
being clear about what case we're producing is a useful statement, rather than stating we match someone elses API.

Reply:
both points addressed.